tests/subscriptions/test_consumer.py

```diff
@@ -0,0 +1,45 @@
+from typing import cast
+
+from snuba.subscriptions.consumer import Tick, TickConsumer
+from snuba.utils.streams.consumer import Consumer
+from snuba.utils.streams.dummy import DummyConsumer, epoch
+from snuba.utils.streams.types import Message, Partition, Topic
+from snuba.utils.types import Interval
+
+
+def test_tick_consumer() -> None:
+    topic = Topic("messages")
+
+    consumer = TickConsumer(
+        cast(
+            Consumer[int],
+            DummyConsumer({Partition(topic, 0): [0, 1, 2], Partition(topic, 1): [0]}),
+        )
+    )
+
+    consumer.subscribe([topic])
+
+    # consume 0, 0
```

Comment:
This should eventually assert the return value of `tell` from the inner consumer, but I haven't exposed that from the `Consumer` ABC yet.